microprofile.jdt/org.eclipse.lsp4mp.jdt.tp/org.eclipse.lsp4mp.jdt.target

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Would it be possible to add a p2 repository entry for https://download.eclipse.org/jdtls/milestones/1.14.0/repository/ ? Specifically, just `org.eclipse.jdt.ls.core` is needed. I think this would eliminate having the pom file entry for that repository.

Also by the looks of it.. PDE doesn't support adding the Maven coordinates to the target platform :disappointed:  I was really hoping just adding something like :

**javaConfig.json**
```
{
    "projects": [
        "microprofile.jdt/org.eclipse.lsp4mp.jdt.core",
        "microprofile.jdt/org.eclipse.lsp4mp.jdt.test"
    ],
   "targetPlatform": "microprofile.jdt/org.eclipse.lsp4mp.jdt.tp/org.eclipse.lsp4mp.jdt.target"
}
```

could also build the whole thing in VSCode !

Reply:
Okay, I took out the pom file entry and added it to the target platform